packages/url_launcher/url_launcher/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 5.4.9
```

Comment:
@cyanglaz what is the pattern for bumping plugin versions? I see that in some cases it's a patch, and in other cases it's a minor bump.

Reply:
This should be a patch as there is no public api changes.
A non-breaking public api change would require a minor bump.
https://dart.dev/tools/pub/versioning#semantic-versions

Reply:
do we strictly follow that though?